cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -36,6 +36,8 @@ def cleanup_vm(vm_name, provider_key, provider_mgmt):
 def pytest_generate_tests(metafunc):
     # Filter out providers without provisioning data or hosts defined
     argnames, argvalues, idlist = testgen.infra_providers(metafunc, 'provisioning')
+    if metafunc.function.func_name == 'test_disk_format_select':
```

Comment:
I think this is unlikely, but to avoid name conflicts can you use identity checking here (`metafunc.function is test_dist_format_select`)?